src/initialize_dae.jl

```diff
@@ -287,12 +288,15 @@ function _initialize_dae!(integrator, prob::ODEProblem,
 	alg_u = @view u[algebraic_vars]
 
 	nlequation = (out, x) -> begin
-		alg_u .= x
-		f(tmp, u, p, t)
-		out .= @view tmp[algebraic_eqs]
+    uu = similar(x, axes(integrator.u)); copyto!(uu, integrator.u)
```

Comment:
whitespace is odd here.

Reply:
This file is all using tabs for some reason. I used spaces.